wordpress-mobile/WordPress-iOS
# Draft Extension: Add draft extension for the Jetpack app

Part of #19361 

## Description
Creates `JetpackDraftActionExtension` which is a replica of the `WordPressDraftActionExtension`.

## Testing Instructions

1. Install Jetpack.
2. Go to the system's Photos app.
3. Select a photo.
4. Tap on the share button.
5. Scroll down and select Jetpack's "Save as Draft" option.
6. Select a site then save the image.
7. Open the Jetpack app.
8. Navigate to Drafts list.
9. Make sure the image is correctly saved. 

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

6. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
